autotst/job/orca_submit.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+module load orca/4.0.1
```

Comment:
This is only specific to discovery / for us (which is why I don't like the idea of an `orca_submit.sh` file. The modules should be loaded independently by the user for their own linux system and then they set the `orca` command accordingly.